pkg/scheduler/apis/config/types.go

```diff
@@ -186,6 +186,9 @@ type ElasticQuotaArgs struct {
 
 	// EnableCheckParentQuota check parentQuotaGroups' used and runtime Quota in PreFilter
 	EnableCheckParentQuota *bool
+
+	// DisableRuntimeQuota if true, use max instead of runtime for all checks.
+	DisableRuntimeQuota *bool
```

Comment:
BTW maybe we can define the field named EnableRuntimeQuota, and by default is true. So other codes will like the following:
```go
var limit corev1.ResourceList
if pl.args.EnableRuntimeQuota {
    limit = pl.quotaManager.RefreshRuntime(quotaName)
}
limit = quotav1.Max(quotaInfo.GetMax(), limit)
```

Reply:
done